backend/de.metas.business/src/main/java/de/metas/costrevaluation/callout/M_CostRevaluation.java

```diff
@@ -92,7 +91,7 @@ private void setDocumentNo(final I_M_CostRevaluation costRevaluation)
 			return;
 		}
 
-		final IDocumentNoInfo documentNoInfo = documentNoBuilderFactory
+		final IDocumentNoInfo documentNoInfo = Services.get(IDocumentNoBuilderFactory.class)
```

Comment:
hmm, why do we have to inline the service here?

Btw, just saw that DocumentNoBuilderFactory is an `@Service` so it might be that instead of
```java
private final IDocumentNoBuilderFactory documentNoBuilderFactory = Services.get(IDocumentNoBuilderFactory.class);
```
it would work with
```java
private final IDocumentNoBuilderFactory documentNoBuilderFactory = SpringContextHolder.instance.getBean(IDocumentNoBuilderFactory.class);
```

Reply:
saw that the PR is green, so pls test / integrate it.

let's not do the change I've suggested above. Let's keep it in mind for next time.

Reply:
i had no default arg constructor, that is why i inline it, next time ill apply the change suggested above.